app/src/main/res/values/dimens.xml

```diff
@@ -247,8 +249,10 @@
   <dimen name="drag_drop_grouping_hint_non_split_text_view_margin_top">8dp</dimen>
   <dimen name="drag_drop_container_parent_non_split_margin_end">8dp</dimen>
   <dimen name="drag_drop_container_parent_non_split_margin_start">8dp</dimen>
-  <dimen name="feedback_item_exploration_view_padding_end">8dp</dimen>
-  <dimen name="feedback_item_exploration_view_padding_start">8dp</dimen>
+  <dimen name="feedback_item_exploration_view_padding_end">0dp</dimen>
+  <dimen name="feedback_item_exploration_view_padding_start">0dp</dimen>
+  <dimen name="feedback_text_view_padding_start">8dp</dimen>
+  <dimen name="feedback_text_view_padding_end">8dp</dimen>
```

Comment:
The names of the text view padding should be started with `feedback_item_text_view` to give a more context about its use.